config/crd/bases/getporter.org_agentconfigs.yaml

```diff
@@ -102,6 +102,12 @@ spec:
                   Porter will request when running the Porter Agent. It is used to
                   determine what the storage class will be for the volume requested
                 type: string
+              ttlSecondsAfterFinished:
```

Comment:
This is going to have to be updated in `getporter/deployments` so that they have the updated CRD to deliver that.